build.rs

```diff
@@ -120,6 +120,7 @@ fn compile_probe(rustc_bootstrap: bool) -> bool {
         .arg("--crate-type=lib")
         .arg("--cap-lints=allow")
         .arg("--emit=dep-info,metadata")
+        .arg("--cap-lints=allow")
```

Comment:
Out of curiosity: Isn't this duplicated now because `--cap-lints=allow` is already being set in line 121? Same for https://github.com/dtolnay/proc-macro2/pull/515.

Reply:
Yes, good call. I missed that the same flags were already present but just ordered differently compared to the `anyhow` build script. I'll revert these.